include/megaapi.h

```diff
@@ -13976,7 +14093,25 @@ class MegaApi
          */
         void checkSMSVerificationCode(const char* verificationCode, MegaRequestListener *listener = NULL);
 
+        /**
+         * @brief Requests the contacts that are registered at MEGA (currently verified through SMS)
+         *
+         * The request will return any of the provided contacts that are registered at MEGA, i.e.,
+         * are verified through SMS (currently). Check the unit tests for an example response.
+         *
+         * @param contacts The map of contacts to get registered contacts from
+         * @param listener MegaRequestListener to track this request
```

Comment:
same here - we should name the request type and valid members

Reply:
I think we should describe what `contacts` actually holds (same with the result). But I don't feel like we should describe the query as that's hidden and the client shouldn't need to know.

Reply:
It's common practice to handle the responses in a global listener by switching on the request type (instead of passing a listener to that specific request).  You'll see all (or at least most) of the other commands contain a comment like "The associated request type with this request is MegaRequest::XYZ" followed by a description of the relevant getters.